### PR TITLE
Coding style enforced

### PR DESCRIPTION
**Pull request by curry684**
As discussed in #36

### CODE REVIEW

File: Exception/InvalidStateException.php [hunk 1]
```diff
@@ -1,11 +0,0 @@
-<?php
```
Comment (weaverryan) [inline Exception/InvalidStateException.php]: Could we keep everything at the root - so not in `src/`? That would keep it consistent, at least, with other bundles.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+*.gz binary
+*.zip binary
+*.7z binary
+*.ttf binary
+*.eot binary
+*.woff binary
+*.pyc binary
+*.pdf binary
+*.woff binary
+*.woff2 binary
```
Comment (weaverryan) [inline .gitattributes]: What's this file based off of? It seems like it's got a lot of unnecessary details

File: Exception/InvalidStateException.php (hunk 1 above)
Comment (weaverryan) [inline Exception/InvalidStateException.php]: Hmm, actually, let's keep them in `src/` - it's indeed a bit more organized.

**Comment by weaverryan** [comment]
Hey @curry684!

I asked just one question about the gitattributes. But this is really really clean - thanks a lot! I'll merge as soon as we resolve that last item.

Cheers!

**Comment by weaverryan** [comment]
And, I want to tag a new release - I'll do it after this.

**Comment by curry684** [comment]
I chose to reorganize the folders because having a bin and tests folder makes it messy. Without those I agree it wouldn't have real added value, now it isolates the code namespace to its own folder.

As for the gitattributes it's a lesson hard learnt on my end after once wasting 2 hours trying to find a file corruption issue to find out it was git itself mistaking a PNG for plaintext and converting 'newlines' before push. Since then I always have it in my project templates, even if actually irrelevant like here, just to avoid running into the same issue later on hehe. I have no problem removing it, it doesn't have real added value right now nor will it likely have that one day for this bundle.

**Comment by weaverryan** [comment]
Thanks @curry684! And I've just tagged a new release :)

**Pull request merged**